website/pages/playground/index.html

```diff
@@ -192,47 +189,27 @@
       }
     }
 
-    .options-details {
-      padding: 8px 0;
+    .options-container {
+      max-height: 100%;
+      overflow-y: scroll;
```

Comment:
Could you change this to `overflow-y: auto;`. It looks strange to have a disabled scroll bar when there’s no overflow (that’s what it looks like on Windows and Linux).

Reply:
Yeah I do, thanks for noticing